com.lunarclient.Launcher.appdata.xml

```diff
@@ -0,0 +1,594 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  
+  <id>com.lunarclient.Launcher</id>
+  
+  <metadata_license>CC0-1.0</metadata_license>
+  
+  <name>Lunar Client</name>
+  
+  <summary>A modpack for all modern versions of Minecraft!</summary>
+  
+  <description>
+    <p>
+      A modpack for all modern versions of Minecraft!
+    </p>
+  </description>
+  
+  <categories>
+  	<category>Game</category>
+  </categories>
+  
+  <url type="homepage">https://lunarclient.com</url>
+  <url type="help">https://support.lunarclient.com</url>  
+  <url type="faq">https://www.lunarclient.com/faq</url>
+
+  <launchable type="desktop-id">com.lunarclient.Launcher.desktop</launchable>
+
+  <releases>
+  	<release version="2.9.3" date="2021-12-03">
+            <description>
+              <p>Update</p>
+              <p>
+              https://www.lunarclient.com/news/patch-notes-17/
```

Comment:
appstream validate will not like bare URL. URL should be in the `<url></url>` tag instead. (you can skip having a `<desription>`)